srcpkgs/mesa/template

```diff
@@ -23,8 +23,9 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT, LGPL-2.1-or-later"
 homepage="https://www.mesa3d.org/"
 changelog="https://docs.mesa3d.org/relnotes.html"
-distfiles="https://mesa.freedesktop.org/archive/mesa-${version}.tar.xz"
-checksum=e641ae27191d387599219694560d221b7feaa91c900bcec46bf444218ed66025
+#distfiles="https://mesa.freedesktop.org/archive/mesa-${version}.tar.xz"
+distfiles="https://mesa3d.org/archive/mesa-${version}.tar.xz"
```

Comment:
Why did you change the distfiles URL?

Reply:
Ok, tested it here. The old link can simply be removed entirely, no reason to leave it commented out.